bibtex-completion.el

```diff
@@ -412,24 +427,32 @@ Also sets `bibtex-completion-display-formats-internal'."
             (user-error "Bibliography file %s could not be found" file)))
             (bibtex-completion-normalize-bibliography))
 
-  ;; Pre-calculate minimal widths needed by the format strings for
-  ;; various entry types:
   (setq bibtex-completion-display-formats-internal
```

Comment:
I was also just wondering: couldn't this be set within the format-entry function?

Reply:
I did a quick comparison of the existing bibtex-completion-format-entry, and my modified one.

``` elisp
ELISP> (benchmark-run 100
  (bibtex-actions--format-entry
   "aclu2011"
   (1- (frame-width))
   bibtex-actions-display-template))
(0.008320599000000001 0 0.0)

ELISP> (benchmark-run 100
  (bibtex-completion-format-entry
   "aclu2011"
   (1- (frame-width))))
(0.027423597 0 0.0)
```

So they're both fast, but using this approach is even faster.

Interestingly, my `bibtex-actions--get-candidates` function, which transforms the output of `bibtex-completion-candidates`, is _really_ slow by comparison, even though the UI in selectrum seems fast.

``` elisp
ELISP> (benchmark-run 100 (bibtex-completion-candidates))
(1.018316332 2 0.2629074330000023)

ELISP> (benchmark-run 100 (bibtex-actions--get-candidates))
(20.117633309 43 6.017887700999999)
```